packages/echo-core/src/services/analytics/appInsightWrapper.ts

```diff
@@ -47,4 +47,10 @@ class AppInsightsWrapper {
     }
 }
 
-export const appInsightsInstance = new AppInsightsWrapper(key);
+let appInsightsWrapper: AppInsightsWrapper | undefined = undefined;
+
+export function appInsightsInstance(): AppInsightsWrapper {
```

Comment:
Some test for this as well? If it really returns the same instance, when called twice.